src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -174,14 +178,17 @@ export default React.memo(
         withOnyx({
             fullReport: {
                 key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+                initialValue: {},
```

Comment:
Why have you added empty object as initial value to each resource here?

Reply:
@sebryu Onyx re-renders the component sometimes if `initialValue` is not provided. You can treat it as a default value.